### PR TITLE
Add config API endpoint

### PR DESCRIPTION
**Pull request by rupurt**
[Finishes #158084312]

### CODE REVIEW

File: store/config.go [hunk 1]
```diff
@@ -168,6 +122,19 @@ type Duration struct {
 	time.Duration
 }
 
+// MarshalText returns the byte slice of the formatted duration e.g. "500ms"
+func (d Duration) MarshalText() ([]byte, error) {
```
Comment (dimroc) [inline store/config.go]: Aren't `MarshalText()` and `UnmarshalText()` for free since we embed `time.Duration`?

File: web/router.go [hunk 2]
```diff
@@ -73,6 +73,9 @@ func serverEngine(app *services.ChainlinkApplication) *gin.Engine {
 		v2.GET("/backup", backup.Show)
 	}
 
+	cc := ConfigController{app}
+	engine.GET("/config", cc.Show)
```
Comment (dimroc) [inline web/router.go]: I personally would version config `/v2/config`

Comment (rupurt) [reply web/router.go]: 👍 I originally thought the same thing but the story asks for `routing: GET request to /config`. Thoughts @smartcontractkit/core-devs?

Comment (dimroc) [reply web/router.go]: We got the greenlight for `/v2/config`, let's change it.

File: store/config.go (hunk 1 above)
Comment (rupurt) [reply store/config.go]: It results in `"databasePollInterval":{"Duration":500000000}` vs the current implementation of `"databasePollInterval":"500ms"`

**Pull request merged**